content/en/docs/concepts/security/pod-security-standards.md

```diff
@@ -0,0 +1,325 @@
+---
+reviewers:
+- tallclair
+title: Pod Security Standards
+content_template: templates/concept
+weight: 10
+---
+
+{{% capture overview %}}
+
+Security settings for pods are typically applied by using [security
+contexts](/docs/tasks/configure-pod-container/security-context/). Security Contexts allows for the
+definition of privilege and access controls on a per-pod basis.
+
+The enforcement and policy-based definition of cluster requirements of security contexts has
+previously been achieved using [Pod Security Policy](/docs/concepts/policy/pod-security-policy/). A _Pod Security Policy_
+is a cluster-level resource that controls security sensitive aspects of the pod specification.
+
+However, numerous means of policy enforcement have arisen that augment or replace the use of
+PodSecurityPolicy. The intent of this page is to detail recommended pod security profiles, decoupled
+from any specific instantiation.
+
+{{% /capture %}}
+
+{{% capture body %}}
+
+## Policy Types
+
+There is an immediate need for base policy definitions to broadly cover the security spectrum. These
+should range from highly restricted to highly flexible:
+
+- **_Privileged_** - Unrestricted policy, providing the widest possible level of permissions. This
+  policy allows for known privilege escalations.
+- **_Baseline/Default_** - Minimally restrictive policy while preventing known privilege
+  escalations. Allows the default (minimally specified) pod configuration.
+- **_Restricted_** - Heavily restricted policy, following current pod hardening best practices.
+
+## Policies
+
+### Privileged
+
+The Privileged policy is purposely-open, and entirely unrestricted. This type of policy is typically
+aimed at system- and infrastructure-level workloads managed by privileged, trusted users.
+
+The privileged policy is defined by an absence of restrictions. For blacklist-oriented enforcement
+mechanisms (such as gatekeeper), the privileged profile may be an absence of applied constraints
+rather than an instantiated policy. In contrast, for a whitelist oriented mechanism (such as Pod
+Security Policy) the privileged policy should enable all controls (disable all restrictions).
+
+### Baseline/Default
+
+The Baseline/Default policy is aimed at ease of adoption for common containerized workloads while
+preventing known privilege escalations. This policy is targeted at application operators and
+developers of non-critical applications. The following listed controls should be
+enforced/disallowed:
+
```

Comment:
Note: There is a `table` shortcode for adding a table caption.
https://kubernetes.io/docs/contribute/style/hugo-shortcodes/#table-captions

Reply:
The table is rendering oddly (styling of lists?). I have not investigated the details.
I _think_ you can include HTML lists in a Markdown table.
It would take time to convert the table to Markdown.

Reply:
Hmm, the styling has to do with CSS rules that are applied to `<p>` and `<li>` elements overriding the styling rules for table content, hence the mixed sizes. This rendered a lot better in github markdown.

I see 2 options, LMK if you can thinking of anything else:
1. Scrap the tables, just make the control column into section headers, and put everything into normal paragraphs and lists.
2. Add some custom CSS rules to override the styles for the nested elements to make them more consistent with the table style.
3. Add some custom CSS to override the table styles to make them more consistent with paragraph styles (why is the table font size so small anyway?)

I'm leaning towards option 1 since this page is already a big mess of custom HTML and that would clean it up. It's too bad though, since I think the table format is easier to digest.

Reply:
@tallclair ,
The content seems to work well with tables.
The docs site is in the process of adding a new theme (new parser recently added) and with that some styling changes. The default styling of tables needs to be addressed.
In the meantime, could you create the table with table data/row elements (column span if needed)?
I totally agree with you about the table data font size.

Reply:
> In the meantime, could you create the table with table data/row elements (column span if needed)?

Sorry, could you clarify? I'm not sure what you mean here.

Reply:
Could you create the table without the lists? 
The display may improve.
Styling (from bootstrap) would help visually, though I am hesitant to add styling (table border, border for rows?) just for this table.
```<div class="table-responsive">
   <table class="table table-bordered"><thead class="thead-light">
```